lib/ui5Framework/npm/Installer.js

```diff
@@ -79,9 +79,22 @@ class Installer {
 						await rimraf(stagingDir);
 					}
 
+					// Check whether target dir already exists and remove it.
+					// A target directory already existing but missing a package.json should
+					// never happen. However, we want to be *really* sure that there is no target
+					// directory so that the rename operation won't have any no trouble.
```

Comment:
Just as a note: This is especially important to fix the broken setup with CLI v2.9.0.

Reply:
Right 👍